### DISCUSSION TITLE
I am unable to get github's student developer packages

### DISCUSSION THREAD

**Discussion by ghost** [community/community#131964] [GitHub Education]
### Hi everyone,

I have given all my legal student proofs.I tried all the proof types with the same things as a proof but still I again and again rejected I tried use my student id card photo as a proof last time on 11/7/2024 but they rejected it. Now I try again with my id card photo on 12/7/2024. I am waiting for the result.

**Comment by krikera** [comment]
Same problem happened with me too. I just waited and re-tried.
